certificate-policy.md

```diff
@@ -65,82 +70,82 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
-
+This Certificate Policy (CP) and requirements for U.S. Federal Government is limited to Publicly Trusted Device Certificates used for identifying and authenticating services accessible on the Internet. 
 
 ### 1.4.2 Prohibited Certificate Uses
-All Person identity certificates including but not limited to Person certificates used for digital signature, S/MIME, person authentication, and encryption.
+All identity certificates for natural persons are not allowed under this policy and the Federal Public Device PKI, including but not limited to identity certificates used to identify natural persons for digital signatures, S/MIME, client authentication, and encryption.  CAs may not issue Subscriber certificates for natural persons or enter into any cross-certification with any CAs that issue certificates used to identify and authenticate natural persons.  
 
 ## 1.5 Policy administration
-This Certificate Policy for the Issuance and Management of Publicly-Trusted Certificates includes criteria established by the CA/Browser Forum for use by Certification Authorities when issuing, maintaining, and revoking publicly-trusted Certificates. This Certificate Policy also includes criteria established by the U.S. Federal Public Key Infrastructure to comply with U.S. Federal Government requirements for U.S. Federal Government Agencies.   This CP may be revised from time to time, as appropriate, in accordance with procedures adopted by the CA/Browser Forum and/or the Federal Public Key Infrastructure.
-
+The Federal Public Key Infrastructure Policy Authority (FPKIPA) manages this policy and represents the interests of the  U.S. Federal CIOs and CISOs.   
 
 ### 1.5.1 Organization Administering the Document
-The Federal Public Key Infrastructure Policy Authority (FPKIPA) is a group of U.S. Federal Government Agencies and is chartered by the U.S. Federal CIO Council.  The FPKIPA owns this policy and represents the interest of the Federal CIOs.  The FPKIPA is responsible for:  
+The Federal Public Key Infrastructure Policy Authority (FPKIPA) is responsible for:  
 * Maintaining this CP, and
 * Approving the CPS for each CA that issues certificates under this policy, and
 * Approving the compliance audit report for each CA issuing certificates under this policy, and
 * Ensuring continued conformance of each CA that issues certificates under this policy with applicable requirements as a condition for allowing continued participation, and
 * Ensuring compliance with CAB Forum Baseline Requirements
 
+The Federal PKI Policy Authority SHALL review and update this Certificate Policy at least every 365 days to ensure compliance with CAB Forum Baseline requirements.  After review and approval, the document version number and a dated changelog entry shall be added even if no changes were deemed necessary. 
```

Comment:
Since CA/B ballots have an expected time-to-implement of much less than 365 days, I think we might want to say: 

> The Federal PKI Policy Authority SHALL review and update this Certificate Policy at least every 365 days, and within X days of any new CAB Forum Baseline requirements.

Reply:
I'm hesitant to put "within X days" because not all CAB Forum Baseline requirement updates may impact or apply.  For example: delegated third party updates wouldn't apply to **every** CA infrastructure for even non-profit or commercial CAs.

For traditional FPKI - we have a 30 day timeframe to incorporate FPKI change proposals that are accepted and voted upon into the policies, with implementation timeframes applied.  

For CA/B requirements, we've looked at every update for the past 2++ years and tracked the requirement changes as 'applicable' or 'not applicable'.   Can we consider a statement that is a bit broader than within X days of **any** new CAB Forum requirement?  Since the policy applies to the root and issuing CAs in a **hierarchical** PKI, the policy and **practices / CPS** need to be updated in tandem.

Reply:
I moved this to Section 2.0 - there were duplicates stated over and over ...added language for when to update and flowdowns? 

https://github.com/uspki/policies/pull/147/commits/60c2d521b1bcbae27540118e060d870e50019c50

Reply:
:+1: